### ISSUE TITLE
Drag-and-drop functionality for reordering to-do items

### ISSUE THREAD

**Issue by mdfekete** [SeanECogan/todo-it#1]
I'd love to be able to drag-and-drop items in my to-do list to reorder them. 

It took me a second to figure out how to do it with the up/down arrows. I originally thought the arrows were meant for scrolling the content in the textarea.

**Comment by SeanECogan** [comment]
Agreed! The arrows are a little unwieldy. But now that I have the Angular code for re-ordering, updating the actual interface for it shouldn't be too bad.
